RecoTracker/LSTCore/src/alpaka/PixelTriplet.h

```diff
@@ -760,8 +760,7 @@ namespace SDL {
     float z1 = zPix[1] / 100;
     float r1 = rtPix[1] / 100;
 
-    float Bz = SDL::magnetic_field;
-    float a = -0.299792 * Bz * charge;
+    float a = -1000/SDL::kR1GeVf * charge;
```

Comment:
The issue is that it should be `100` instead of `1000`.

Reply:
Thank you for spotting this! Bugs fixed